forage-android/src/main/java/com/joinforage/forage/android/core/services/forageapi/network/error/payload/SingleErrorResponse.kt

```diff
@@ -0,0 +1,34 @@
+package com.joinforage.forage.android.core.services.forageapi.network.error.payload
+
+import com.joinforage.forage.android.core.services.forageapi.network.error.ForageErrorDetails
+import org.json.JSONObject
+
+/**
+ * Converts error responses passed as single errors to a [Failure] instance.
+ * For example:
+ *
+ * {
+ *   "ref": "e1fff94f29",
+ *   "balance": null,
+ *   "content_id": "c1898593-fa3d-4a1c-b16b-10ecc38b3619",
+ *   "error": {
+ *     "message": "Invalid card number - Re-enter Transaction",
+ *     "forage_code": "ebt_error_14",
+ *     "status_code": 400
```

Comment:
Are we losing the status code now?

Reply:
Excellent question! I updated the names of these classes to have the suffix `Payload` to clarify that they only parse the payload of the error response. `ForageError` and `ForageApiResponse.Failure` still contain the status code. It's just that the status code is not a concern of parsing the error payload